test/arquillian-extension/src/main/java/io/narayana/lra/arquillian/spi/NarayanaLRARecovery.java

```diff
@@ -19,10 +22,26 @@
 
 public class NarayanaLRARecovery implements LRARecoveryService {
     private static final Logger log = Logger.getLogger(NarayanaLRARecovery.class);
+    private static final long WAIT_CALLBACK_TIMEOUT = initWaitForCallbackTimeout();
+    private static final String WAIT_CALLBACK_TIMEOUT_PROPERTY = "lra.tck.callback.timeout";
 
+    /*
+     * Wait for the participant to return the callback. This method does not
+     * guarantee callbacks to finish. The Participant status is not immediately
+     * reflected to the LRA status, but only after a recovery scan which executes an
+     * enlistment. The waiting time can be configurable by
+     * LRAConstants.WAIT_CALLBACK_TIMEOUT property
+     */
     @Override
     public void waitForCallbacks(URI lraId) {
-        // no action needed
+        log.info("Wait for the callback to be returned by the CompletionStage");
+        log.info("waitForCallbacks for: " + lraId.toASCIIString());
```

Comment:
Unnacessary and it will span all the logs in tests. Remove please.

Reply:
I would rather keep one log and set it to trace level. Thanks